Tests/Recurly/Plan_Test.php

```diff
@@ -122,31 +146,14 @@ public function testUpdateXml() {
     );
   }
 
-  public function testUpdateWithRampsXml() {
```

Comment:
Removed this since it just tests serialization which is covered in the `testCreateWithRampsXml` test case.